lib/jeff.ex

```diff
@@ -174,6 +176,79 @@ defmodule Jeff do
     ACU.send_command(acu, address, MFG, params) |> handle_reply()
   end
 
+  @doc """
+  Send file data to a PD
+  """
+  @spec file_transfer(acu(), osdp_address(), binary()) ::
+          Reply.FileTransferStatus.t() | cmd_err()
+  def file_transfer(acu, address, data) when is_binary(data) do
+    file_transfer(acu, address, data, nil)
```

Comment:
```suggestion
    print_progress_callback = 
      fn num, total, percent ->
        {:ok, cols} = :io.columns()
        total_cols = round(cols * 0.7)
        progress_len = round(total_cols * (num / total))
        progress = :binary.copy("=", progress_len)
        trailing = :binary.copy(" ", total_cols - progress_len)
        result = "\r[#{progress}#{trailing}] #{num}/#{total} (%#{percent})"
        if num == total, do: IO.puts(result), else: IO.write(result)
      end
    file_transfer(acu, address, data, print_progress_callback)
```
Would an IO printing default be good here for when a callback isn't specified?

Reply:
I like what you've added her.